Base/usr/share/man/man1/basename.md

```diff
@@ -0,0 +1,28 @@
+## Name
+
+basename - print basename of file
```

Comment:
Someone who doesn't know what basename does wouldn't know what "print basename of file" means.